int32 mode
geometry_msgs/PoseStamped desired_pose
---
